widgetschema.json

```diff
@@ -115,6 +115,10 @@
                         }
                     },
                     "description": "Mapping of the API result to the choices"
+                },
+                "mappedValues": {
+                    "type": "object",
+                    "description": "Values from other widget properties that will replace parts of the url. The key specifies the url segment to replace, and the value indicates the corresponding property name."
```

Comment:
Couple questions:

1. Will this be able to access properties inside input groups, e.g. `someInputGroup.somePropInTheInputGroup`?
2. If the `choices` property is inside a `multiple`, does the value reference here relative to the current item inside the `multiple` or is it relative to the root widget properties?  Or preferably, is there a way to get to either one depending on what's desired?